### ISSUE TITLE
Vagrant lost it's association with existing boxes

### ISSUE THREAD

**Issue by hillar** [mitchellh/vagrant#7091]
There seem to be a number of similar issues:  GH-5105 etc..

``` bash
$ vagrant -v
Vagrant 1.8.1

$ vagrant plugin list
vagrant-vsphere (1.6.0)

$ uname -a
Darwin student.local 15.2.0 Darwin Kernel Version 15.2.0: Fri Nov 13 19:56:56 PST 2015; root:xnu-3248.20.55~2/RELEASE_X86_64 x86_64
```
----

``` bash

$ vagrant --provider=vsphere halt
==> student-1-suricata-2: The VM has not been created
==> student-1-suricata-1: The VM has not been created

$ vagrant --provider=vsphere up
Bringing machine 'student-1-suricata-1' up with 'vsphere' provider...
Bringing machine 'student-1-suricata-2' up with 'vsphere' provider...
==> student-1-suricata-2: Calling vSphere CloneVM with the following settings:
==> student-1-suricata-1: Calling vSphere CloneVM with the following settings:
==> student-1-suricata-2:  -- Template VM: vm/CMDCS/Templates/cmdcs1clust2nic2
==> student-1-suricata-1:  -- Template VM: vm/CMDCS/Templates/cmdcs1clust2nic2
==> student-1-suricata-2:  -- Target VM: vm/CMDCS/STUDENTS/Student_1/student-1-suricata-2
==> student-1-suricata-1:  -- Target VM: vm/CMDCS/STUDENTS/Student_1/student-1-suricata-1
==> student-1-suricata-2: An error occurred. The error will be shown after all tasks complete.
==> student-1-suricata-1: An error occurred. The error will be shown after all tasks complete.
An error occurred while executing multiple actions in parallel.
Any errors that occurred are shown below.
An error occurred while executing the action on the 'student-1-suricata-1'
machine. Please handle this error then try again:
DuplicateName: The name 'student-1-suricata-1' already exists.
An error occurred while executing the action on the 'student-1-suricata-2'
machine. Please handle this error then try again:
DuplicateName: The name 'student-1-suricata-2' already exists.


$ vagrant --provider=vsphere status 
Current machine states:
student-1-suricata-1      vagrant_vsphere.states.short_not_created (vsphere)
student-1-suricata-2      vagrant_vsphere.states.short_not_created (vsphere)

$ vagrant global-status 
9355825  student-1-suricata-1    vsphere    vagrant_vsphere.states.short_running          git/CDMCS/suricata/day_1/boxes   
f51fb72  student-1-suricata-2    vsphere    vagrant_vsphere.states.short_running          git/CDMCS/suricata/day_1/boxes   

```

**Comment by sethvargo** [comment]
Hi @hillar 

Thank you for opening an issue. Unfortunately the Vagrant vsphere provider is a third-party plugin, and we are unable to support third-party plugins in Vagrant core. If you are able to reproduce this using just virtualbox, please open a new issue and follow the issue template so we can try to reproduce it. Thanks and sorry! :smile:

**Comment by hillar** [comment]
it is not about plugin, it is about vagrant.
plugin works - see errors from *up* command
it is vagrant *'local decision:': short_not_created* and it never goes to really check, is  vm there or not.
please reopen

**Comment by sethvargo** [comment]
Hi @hillar 

All of that output and the commands that are run under the hood are part of the vsphere provider plugin, not Vagrant. Please open an issue on that provider.
